variables.tf

```diff
@@ -595,3 +595,13 @@ variable "admin_user" {
   type        = string
   default     = "kong-admin"
 }
+
+variable "lb_logging_bucket" {
```

Comment:
Oh my mistake! we need to et the defaults to `lb_logging_bucket` and `external_lb_logging_prefix` to `""`

Reply:
https://github.com/faros-ai/terraform-aws-kong/pull/10